tests/_test_mscolab/test_server.py

```diff
@@ -80,7 +80,9 @@ def test_hello(self):
         with self.app.test_client() as test_client:
             response = test_client.get('/status')
             assert response.status_code == 200
-            assert b"Mscolab server" in response.data
+            data = json.loads(response.text)
+            assert "Mscolab server" in data['message']
+            assert True or False in data['IDP_ENABLED']
```

Comment:
The config var needs to be added into the mscolab_settings.py which gets created by
https://github.com/Open-MSS/MSS/blob/GSOC2023-NilupulManodya/conftest.py#L111
on each test run. 

The more interesting question is, why this test doesn't fail?